### PR TITLE
EUI-3473: Challenged Access Request screen for Case Access Management

### PR DESCRIPTION
**Pull request by Daniel-Lam**
### JIRA link (if applicable) ###
https://tools.hmcts.net/jira/browse/EUI-3473

### Change description ###
Implementation of display-only User Story.

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

**Review by rayliangatos — APPROVED** [review]
looks good, approved.

File: src/shared/components/case-viewer/viewer.routing.ts [hunk 1]
```diff
@@ -23,5 +24,9 @@ export const viewerRouting: Routes = [
     {
       path: 'event/:eid/history',
       component: CaseHistoryComponent,
+    },
+    {
+      path: 'access-request',
```
Comment (adnanakgun) [inline src/shared/components/case-viewer/viewer.routing.ts:29]: Do you think path should have the word `challenged` in it? Just a suggestion.

**Review by adnanakgun — DISMISSED** [review]
Approved with a suggestion

File: src/shared/components/case-viewer/viewer.routing.ts (hunk 1 above)
Comment (Daniel-Lam) [reply src/shared/components/case-viewer/viewer.routing.ts:29]: @adnanakgun - could be an idea... although, I'm not sure if we're going to reuse this component for the "Specific Access" case - in which case, they _may_ want to use the same route... perhaps.

**Review by adnanakgun — APPROVED** [review]

**Review by rayliangatos — APPROVED** [review]

**Pull request merged**